app/models/event.rb

```diff
@@ -41,6 +41,7 @@ class Event < ActiveRecord::Base
   scope :canceled, -> { where(state: 'canceled') }
   scope :withdrawn, -> { where(state: 'withdrawn') }
   scope :highlighted, -> { where(is_highlight: true) }
+  scope :current, -> { joins(:event_type).where("start_time <= ? AND datetime(strftime('%s', start_time) + event_types.length * 60,  'unixepoch') >= ?", Time.current, Time.current) }
```

Comment:
Please note that https://github.com/openSUSE/osem/pull/1094 will change this assocation/attributes. Depending on which PR we will merge first, we need to adapt the other one!

^^ @Ana06